### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by geopayme** [Pakcoin-project/pakcoinold#1]
Any interest to have Pakcoin included in the UberPay Bitcoin/Multicoin wallet family of Altcoins?

**Comment by drwatson84** [comment]
Please contact on admin@pakcoin.info

**Comment by drwatson84** [comment]
Hi
What's your offer?
Contact at admin@pakcoin.info

**Comment by geopayme** [comment]
We are not selling anything, our goal is to build a simple service that helps consumers and merchants adapt coins better. See http://uberpay.io

**Comment by drwatson84** [comment]
Ok
So how could I include pakcoin into Uber pay wallet.
You can guide me for this.
Thanks

**Comment by geopayme** [comment]
We will create a profile for Pakcoin similar to the one below... with the coin values

public class BitcoinMain extends CoinType {
    private BitcoinMain() {
        id = "bitcoin.main";

```
    addressHeader = 0; // public key address
    p2shHeader = 5;    // script address
    acceptableAddressCodes = new int[] { addressHeader, p2shHeader };
    spendableCoinbaseDepth = 100;
    dumpedPrivateKeyHeader = 128;

    family = BitFamily.get();
    name = "Bitcoin";
    fullname = "Bitcoin";
    symbol = "BTC";
    uriScheme = "bitcoin";
    bip44Index = 0;
    unitExponent = 8;
    feePerKb = value(10000);
    minNonDust = value(5460);
    softDustLimit = value(1000000); // 0.01 BTC
    softDustPolicy = SoftDustPolicy.AT_LEAST_BASE_FEE_IF_SOFT_DUST_TXO_PRESENT;
}
```

One of the critical items needed is a Pakcoin Electrum server that can be setup by the Pakcoin Dev team and the rest is history and marketing.

You can download the UberPay wallet from Google's play store to evaluate it.

Cheers

**Comment by drwatson84** [comment]
After crating pakcoin electrum server, you would connect the app to the
server, did I get right?
Alright we would try to set that up as soon as we get time.
Thanks

**Comment by geopayme** [comment]
That is correct, I can work on the code and let you know. Thanks

**Comment by drwatson84** [comment]
Hi
I have configured Pakcoin electrum server at 108.61.246.159 with default
ports, please check and let me know if it is working at your end ?

Thanks
please reply to admin@pakcoin.info

**Comment by geopayme** [comment]
Can you tell me what port are you using?

**Comment by drwatson84** [comment]
Ports are:
electrum_rpc_port = 8000
stratum_tcp_port = 50001

**Comment by drwatson84** [comment]
Ports are:
electrum_rpc_port = 8000
stratum_tcp_port = 50001

**Comment by geopayme** [comment]
The Pak server is off line 

Monitored Servers

Host    Port    Proto   UTXO Root   Height  Blocktime   Version Connection  ConnectionTime  Status  Uptime Hour Day Month
108.61.246.159  50001   tcp     0           closed  2015-12-05 04:56:13 CLOSED  0.0000 0.0000 0.0000

**Comment by geopayme** [comment]
I send a followup on admin@pakcoin.info
